plugins/by-name/easy-dotnet/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib, config, ... }:
+lib.nixvim.plugins.mkNeovimPlugin {
+  name = "easy-dotnet";
+  packPathName = "easy-dotnet.nvim";
+  package = "easy-dotnet-nvim";
+
+  maintainers = [ lib.maintainers.GaetanLepage ];
+
+  settingsExample = {
+    test_runner = {
+      enable_buffer_test_execution = true;
+      viewmode = "float";
+    };
+    auto_bootstrap_namespace = true;
+    terminal.__raw = ''
+      function(path, action, args)
+        local commands = {
```

Comment:
the docs will render this horribly:

```
terminal = {
  __raw = "function(path, action, args)\n  local commands = {\n";
}
```

This is because `lib.generators.toPretty` will use `builtins.toJSON` to render any string values.

I'd recommend using a literalExpression for the example this time